.github/workflows/test.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         run: node_modules/.bin/mocha --grep 'e2e.*' -i --exit
       - name: Login
         run: checkly-cli login --api-key ${{ secrets.USER_API_KEY_TEST }} --account-id ${{ secrets.USER_ACCOUNT_TEST }}
```

Comment:
I guess we should now delete this line?

Reply:
Yeah something went wrong with my commit, I just changed the name of the variables in my editor haha. 1 sec